modules/logbucket/main.tf

```diff
@@ -40,6 +40,9 @@ resource "google_logging_project_bucket_config" "bucket" {
   enable_analytics = var.enable_analytics
   bucket_id        = var.name
   locked           = var.locked
+  cmek_settings {
```

Comment:
This code will be optional and should execute when keys are provided. Check [storage module](https://github.com/terraform-google-modules/terraform-google-log-export/blob/fd276c307245d76f4b136e00df60572458b19316/modules/storage/main.tf#L74) how to handle it?

Reply:
@imrannayer should we add a new folder with these settings as an option?